src/styles/index.scss

```diff
@@ -1,3 +1,141 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, serif;
+}
+
+p {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
```

Comment:
Consider using SCSS variables for colors and other main values instead of hardcoding them. This will make it easier to manage and update styles.